modules/aws_asg/kill-old-containers.bash

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+__die() {
+  local status="${1}"
+  local code="${2}"
+  local count="${3}"
+  logger "time=$(date -u +%Y%m%dT%H%M%S) " \
+    "prog=$(basename "${0}") status=${status} count=${count}"
+  exit "${code}"
+}
+
+main() {
+  local worker_cid=$(docker inspect travis-worker --format '{{ .Id }}')
+  local cids=$(docker ps -q --filter before="$worker_cid")
```

Comment:
Sorry, I didn't explain this well 😅   Shellcheck gets upset about locals defined in-line via subshell due to issues with exit code handling, so splitting up the declarations and assignments is needed, a la:

``` bash
local somefoo
somefoo="$(docker inspect foo --format '{{ .Id }}')"
```

That's what these errors are about, at least: https://travis-ci.org/travis-infrastructure/terraform-config/builds/308032321#L434-L441